sunpy/visualization/colormaps/color_tables.py

```diff
@@ -87,6 +87,10 @@ def aia_color_table(wavelength):
 
     Based on aia_lct.pro part of SDO/AIA on SSWIDL written by Karel
     Schrijver (2010/04/12).
+
+    Parmeters
+    ---------
+    wavelength : int
```

Comment:
```suggestion
    wavelength : `int`
        The wavelength  for the color table you are after. 
```
Maybe?

Reply:
Slightly out of scope, but it bothers me that this is an `int` and not a `u.Quantity`

Reply:
Yeah, it should probably be a quantity.

Reply:
I'll just move this to a separate issue